README.md

```diff
@@ -1,73 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# NFT CONFERENCE-CAPSTONE
 
+> A capstone Project template for an Nft Global Conference
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- HTML
+- CSS
+- Javascript
+- SASS
+- Bootstrap
 
-To get a local copy up and running follow these simple example steps.
+## Live Demo
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+[Live Demo](https://promzzykoncepts.github.io/NFT-Conference/)
 
+## Video Demo
 
+[Video Demo](https://promzzykoncepts.github.io/NFT-Conference/)
 
```

Comment:
hello again @mukhammadrizooff 
I STRONGLY DISAGREE WITH THIS REVIEW  because I have already uploaded the video presentation and pushed changes but it seems you missed that part out. 
Kindly re-review